forte/data/data_store.py

```diff
@@ -474,3 +476,79 @@ def prev_entry(self, tid: int) -> List:
             The previous entry of the same type as the ``tid`` entry.
         """
         raise NotImplementedError
+
+    def parse_onto_file(self):
+        r"""If user provides a customized ontology json file, we parse
+        this file and set self._type_attributes accordingly. It not only
+        includes the classes in the provided ontology file, but also import
+        classes from parent entry files.
```

Comment:
First, Consider improving the documentation:

1. Some small grammar issues (e.g. a user)
2. Quote with backtip when referring to variables.
3. Try to explain in a way that people without prior knowledge can understand:
    a. when referring to an internal variable like `self._type_attributes`, explain what it is.
    b. explain the functionality clearly and exact. For example, "provides a customized ontology json" is not very accurate, what do you mean by "custom"? We should say "This function will populate the types and attributes used in the `data_store` with an ontology specification file". Try to avoid ambiguous words like "ontology json file". The same applies to other sentences too. 

Second, in the current design, can the function be private?

Reply:
Something like "customized ontology specification" is still there, the meaning of "custom"  is not clear, I don't think we need to mention anything about "custom", we just need to mention that if a spec is passed we will use it to populate some internal states.

Now it is also unclear where it is passed?